presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddLocalExchanges.java

```diff
@@ -189,6 +190,14 @@ public PlanWithProperties visitSort(SortNode node, StreamPreferredProperties par
             return planAndEnforceChildren(node, singleStream(), defaultParallelism(session));
         }
 
+        @Override
+        public PlanWithProperties visitAnalyzeFinish(AnalyzeFinishNode node, StreamPreferredProperties context)
+        {
+            // analyze commit requires that all data be in one stream
```

Comment:
> analyze commit

do you mean "analyze finish"?

Reply:
I think it was mimicked from the `visitTableFinish`